open3d_conversions/package.xml

```diff
@@ -12,15 +13,12 @@
 
   <author email="matnay17@gmail.com">Pranay Mathur</author>
   <author email="nkhedekar@nevada.unr.edu">Nikhil Khedekar</author>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
 
-  <buildtool_depend>catkin</buildtool_depend>
-  <depend>roscpp</depend>
+  <depend>rclcpp</depend>
   <depend>sensor_msgs</depend>
-  <depend>eigen</depend>
```

Comment:
You still use Eigen3, needs to have a depend

Reply:
Fixed in d09626d